src/Qbead.h

```diff
@@ -273,63 +421,11 @@ class Qbead {
     q = q * q;
 
     uint8_t rc = redch(c);
-    uint8_t bc = bluech(c);
     uint8_t gc = greench(c);
+    uint8_t bc = bluech(c);
```

Comment:
formatting and reordering comment: if moving this is not necessary, please avoid doing it in a "feature" PR, so that git-blame is easy to use in the future.

I also generally prefer the order to be the standard RGB.